src/Query/Builder.php

```diff
@@ -1053,8 +1053,9 @@ protected function compileWheres(): array
             $method = 'compileWhere' . $where['type'];
             $result = $this->{$method}($where);
 
+            // Negate the expression
             if (str_ends_with($where['boolean'], 'not')) {
-                $result = ['$not' => $result];
+                $result = ['$nor' => [$result]];
```

Comment:
I don't see [PHPORM-90](https://jira.mongodb.org/browse/PHPORM-90) linked to anything. This this bug predate us taking over the library, or was it introduced during some recent query refactoring? If the latter, we should link it up.

Reply:
I implemented `whereNot` in https://github.com/GromNaN/laravel-mongodb/pull/13. The issue [PHPORM-49](https://jira.mongodb.org/browse/PHPORM-49) is now linked in Jira.